include/ruby/internal/intern/process.h

```diff
@@ -30,6 +30,16 @@ RBIMPL_SYMBOL_EXPORT_BEGIN()
 
 /* process.c */
 
+/**
+ * Creates a new instance of Process::Status.
+ *
+ * @param[in] pid The process ID.
+ * @param[in] status The process status.
```

Comment:
```suggestion
 * @param[in] status The "waitpid status", as returned by waitpid(2). This is NOT the exit status/exit code, see waitpid(2).
```
i.e. it's not the "exit status"/"exit code" (which can be accessed with `WEXITSTATUS(wstatus)`, how confusing), it's more than that.

Reply:
Fixed in d30aa0c654dd34e666fb9c493b19bbddda4cce40.